task/fbc-validation/0.1/README.md

```diff
@@ -0,0 +1,24 @@
+# fbc-validation task
+
+## Description:
+The fbc-validation task will ensure FBC (File based catalog) components uniquely linted to ensure they're properly
+constructed as part of the build pipeline. To validate the image in build pipeline, Skopeo is used to extract
+information and then contents are checked using the OpenShift Operator Framework.
+
+## Params:
+
+| name         | description                     |
+|--------------|---------------------------------|
+| image-digest | Image digest.                   |
+| image-url    | The fully qualified image name. |
+
+## Params:
+
+| name               | description        |
+|--------------------|--------------------|
+| HACBS_TEST_OUTPUT  | Test output.       |
+
```

Comment:
I think it might be good to also add a link to the source repository, something like this:
```
## Source repository for image:
https://github.com/redhat-appstudio/hacbs-test
```